org.vergecurrency.verge-qt.desktop

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
```

Comment:
Ideally should be shipped in your tarball/archive for others to consume.

Reply:
we can certainly add that to the tar.